src/pages/portfolio/Overview.tsx

```diff
@@ -51,13 +52,11 @@ export const Overview = () => {
                 ]
               : [
                   PositionsTableColumnKey.Market,
-                  PositionsTableColumnKey.Side,
                   PositionsTableColumnKey.Size,
-                  PositionsTableColumnKey.Leverage,
-                  PositionsTableColumnKey.LiquidationAndOraclePrice,
                   PositionsTableColumnKey.UnrealizedPnl,
-                  PositionsTableColumnKey.RealizedPnl,
+                  !testFlags.isolatedMargin && PositionsTableColumnKey.RealizedPnl,
```

Comment:
outta curiosity, what's the rationale behind not showing PnL when isolated margin is enabled?

Reply:
intention is just to reduce columns in the table to make it not overflow!